Update activate-using-active-directory-based-activation-client.md (#9010)
MicrosoftDocs/windows-itpro-docs: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md

1. Activate your KMS host key by phone or online (Figure 15).

    ![Entering your KMS host key](../images/volumeactivationforwindows81-14.jpg)
    
    > [!NOTE]

>     > For activating a KMS Host Key (CSVLK) for Microsoft Office you need to install the version specific Office Volume license pack on the server where you have installed the Volume Activation Server Role.

Reviewer: Grammar improvement & capitalization in "Volume License Pack" (as shown on https://www.microsoft.com/en-us/download/details.aspx?id=57342 & https://www.microsoft.com/en-us/download/details.aspx?id=49164)
```suggestion
    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.

```

1. Activate your KMS host key by phone or online (Figure 15).

    ![Entering your KMS host key](../images/volumeactivationforwindows81-14.jpg)
    
    > [!NOTE]

>     > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.

Reviewer 2: ```suggestion
    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version-specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.

```

>     - [Office 2013 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=35584)

Reviewer 2: ```suggestion
    - [Office 2013 VL pack](https://www.microsoft.com/download/details.aspx?id=35584)

```

>     - [Office 2016 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=49164)

Reviewer 2: ```suggestion
    - [Office 2016 VL pack](https://www.microsoft.com/download/details.aspx?id=49164)

```

>     - [Office 2019 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=57342)

Reviewer 2: ```suggestion
    - [Office 2019 VL pack](https://www.microsoft.com/download/details.aspx?id=57342)

```

Reviewer: Looks like the suggestion above added a not needed blank line. Going to try suggesting the link change without adding that extra blank line.
```suggestion
    - [Office 2019 VL pack](https://www.microsoft.com/download/details.aspx?id=57342)
```

    > [!NOTE]
    > If you are using both KMS and Active Directory-based activation, it may be difficult to see whether a client has been activated by KMS or by Active Directory-based activation. Consider disabling KMS during the test, or make sure that you are using a client computer that has not already been activated by KMS. The **slmgr.vbs /dlv** command also indicates whether KMS has been used.

> > If you would like to see or mass change this, please consider to use [VAMT](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/volume-activation-management-tool).

Reviewer: ```suggestion
> To manage individual activations or apply multiple (mass) activations, please consider using [VAMT](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/volume-activation-management-tool).
```

Reviewer: The line above could possibly be phrased better, but I don't have enough personal experience in this topic to provide a better content for this line.

    > [!NOTE]
    > If you are using both KMS and Active Directory-based activation, it may be difficult to see whether a client has been activated by KMS or by Active Directory-based activation. Consider disabling KMS during the test, or make sure that you are using a client computer that has not already been activated by KMS. The **slmgr.vbs /dlv** command also indicates whether KMS has been used.

> > To manage individual activations or apply multiple (mass) activations, please consider using [VAMT](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/volume-activation-management-tool).

Reviewer 2: ```suggestion
> To manage individual activations or apply multiple (mass) activations, please consider using the [VAMT](https://docs.microsoft.com/windows/deployment/volume-activation/volume-activation-management-tool).

```
